src/Goerli-DssSpell.t.base.sol

```diff
@@ -180,6 +184,9 @@ contract GoerliDssSpellTestBase is Config, DSTest, DSMath {
     LerpFactoryAbstract lerpFactory = LerpFactoryAbstract(addr.addr("LERP_FAB"));
     VestAbstract            vestDai = VestAbstract(       addr.addr("MCD_VEST_DAI"));
 
+    RwaLiquidationLike       oracle = RwaLiquidationLike( addr.addr("MIP21_LIQUIDATION_ORACLE"));
```

Comment:
`oracle` is too general, maybe `liquidationOracle`?
Redundant empty line above this (it should be part of the above list).

Reply:
@nduchak can you also remove the redundant empty line?